Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -295,7 +295,7 @@ - (void)addNotificationListeners
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(yapDatabaseModified:)
                                                  name:YapDatabaseModifiedNotification
-                                               object:nil];
+                                               object:TSStorageManager.sharedManager.dbNotificationObject];
```

Comment:
If we do user-facing backup, we'll probably export (a subset of) the messages, attachments and groups to another instance of `OWSStorage`.  We'd want to ignore any modification events that result.